src/app/components/LiveLabel/Text/index.tsx

```diff
@@ -51,16 +53,28 @@ const Text = ({
       role="text"
       className={className}
     >
-      <span
-        css={styles.liveLabelText}
-        dir={dir}
-        {...(ariaHidden && { 'aria-hidden': 'true' })}
+      <div
+        data-testid="liveLabelContainer"
+        css={
+          isHeaderImage
+            ? styles.liveLabelContainerWithImage
+            : styles.liveLabelContainerWithoutImage
+        }
```

Comment:
I don't think we should be doing style switching within compound components based on passed in prop booleans. Maybe there's better way to override the styling of this div directly from our LiveLabelHeader component?